### DISCUSSION TITLE
how to bypass local domain?

### DISCUSSION THREAD

**Discussion by rain2307** [hiddify/hiddify-app#601] [Q&A] [answered]
I use custom hosts. How can bypass these domains?
Thanks.

**Comment by vortex4242** [comment, marked as answer]
Bypassing a local domain typically involves altering your network settings or using a VPN to access the internet directly rather than through the local domain's network. However, it's important to note that bypassing local domain restrictions may violate your organization's policies or local laws, so proceed with caution and ensure you have proper authorization before attempting to bypass any restrictions.

**Reply by rain2307** [reply]
Thank you. how can I bypass "xxx.com"?
